src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1311,10 +1311,14 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
-  <data name="PosssibleMultipleEnumerationsOfIEnumrableCollection" xml:space="preserve">
+  <data name="AvoidMultipleEnumerationsMessage" xml:space="preserve">
     <value>Possible multiple enumerations of 'IEnumerable' collection. Consider using an implementation that avoid multiple enumerations.</value>
     <comment>{Locked="IEnumerable"}</comment>
   </data>
+  <data name="AvoidMultipleEnumerationsTitle" xml:space="preserve">
+    <value>Possible multiple enumerations of 'IEnumerable' collection</value>
```

Comment:
For me just `Avoid multiple enumerations` is better shorter version, but I am OK to keeping the old title too